core/src/engine/Method_MC.cpp

```diff
@@ -60,10 +61,74 @@ namespace Engine
 
         // TODO: add switch between Metropolis and heat bath
         // One Metropolis step
-        Metropolis(spins_old, spins_new);
+        Parallel_Metropolis(spins_old, spins_new);
```

Comment:
Is the parallel implementation slower than the sequential version, when running on a single thread?
- If so, it may make sense to be able to switch between them either dynamically (depending on `omp_num_threads`) or statically `#ifdef SPIRIT_USE_OPENMP`
- Otherwise, it would make sense to completely replace the `Metropolis` implementation, right? Then there would also be no need to call it `Parallel_Metropolis`.